reproducibility/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM pangwei/wilds_src:1.0
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+# Install dependencies
+RUN pip install -r requirements.txt && \
```

Comment:
Do we need a separate requirements.txt? pip install wilds should handle torch/torchvision/transformers right?

Reply:
Okay, replaced it with just `pip install wilds`.